web/docs/Backend Starters/Decoupled WordPress/configuring-build-hooks.md

```diff
@@ -0,0 +1,57 @@
+---
+id: 'configuring-build-hooks'
+title: 'Configuring Build Hooks'
```

Comment:
Right now the hooks can only build, but in the future they may be able to do other things. Should we be more generic here and call them 'Webhooks'? That way the doc can be "Configuring Webhooks" with a section on how to configure a "build hook".
cc @backlineint

Reply:
I agree that a more generic name might be a better fit but since our UI uses the `Build Hook` naming I think it makes since to keep it consistent to that.